src/azure-cli/azure/cli/command_modules/storage/_params.py

```diff
@@ -97,6 +97,20 @@ def load_arguments(self, _):  # pylint: disable=too-many-locals, too-many-statem
                                                   "Required when --enable-files-adds is set to True")
     sas_help = 'The permissions the SAS grants. Allowed values: {}. Do not use if a stored access policy is ' \
```

Comment:
This line was above line "with self.argument_context('storage') as c:", it is embeded in the argumenttype definition now